opencti-platform/opencti-front/src/private/components/analysis/Reports.tsx

```diff
@@ -0,0 +1,259 @@
+import React, { FunctionComponent } from 'react';
```

Comment:
I can't create a new Report, I have this error

"Cannot read properties of undefined (reading 'getLinkedRecords')"